src/generators/schema-gen.ts

```diff
@@ -19,7 +19,7 @@ const distinct = (array: Array<any>, discriminator: (item: any) => string): Arra
     });
 };
 
-export const generateJsonSchemaWithOptions = (options: any) => (instance: Object): JsonSchema => {
+export const generateJsonSchemaWithOptions = (options: any = {}) => (instance: Object): JsonSchema => {
```

Comment:
would it not make sense to rename this one to generateJsonSchema and remove the old generateJsonSchema?